seqr/views/apis/individual_api.py

```diff
@@ -703,13 +703,18 @@ def _get_record_updates(record, individual, invalid_values, allowed_assigned_ana
             if k == ASSIGNED_ANALYST_COL:
                 if v not in allowed_assigned_analysts:
                     raise ValueError
-                parsed_val = v
+                if v:
+                    update_record[k] = v
             else:
-                parsed_val = INDIVIDUAL_METADATA_FIELDS[k](v)
-                if (k not in {FEATURES_COL, ABSENT_FEATURES_COL} and parsed_val == getattr(individual, k)) or has_same_features:
-                    parsed_val = None
-            if parsed_val:
-                update_record[k] = parsed_val
+                _parsed_val = INDIVIDUAL_METADATA_FIELDS[k](v)
```

Comment:
is there a reason to use `_parsed_val` instead of `parsed_val`? Its pretty outside of the style we use everywhere else in the repo